_data/ph_authors.yml

```diff
@@ -989,3 +989,19 @@
         fr: |
             Sofia Papastamkou est docteure en histoire et ingénieure d'études en humanités numériques au CNRS. Elle est affectée à la Maison européenne des sciences de l'homme et de la société Lille Nord de France.
         
+  
+  - name: Marie Puren
```

Comment:
Thank you for submitting this. Can you please remove the indentation on `- name:`? This should fix the error.

Reply:
That's done and I hope it will fix the error.